README.md

```diff
@@ -0,0 +1,8 @@
+# Memories Keeper
+
+## Firebase
+
+### Auth
+
+Make sure to add the SHA-1 fingerprint to the Firebase console: 
+`keytool -list -v -keystore "%USERPROFILE%\.android\debug.keystore" -alias androiddebugkey -storepass android -keypass android`
```

Comment:
This command didn't work for me. But this one did (with the password `android`):
`keytool -list -v \
-alias androiddebugkey -keystore ~/.android/debug.keystore`